tasks/update-cmr-access-constraints/index.js

```diff
@@ -15,6 +16,50 @@ const {
   uploadUMMGJSONCMRFile,
 } = require('@cumulus/cmrjs/cmr-utils');
 
+/**
+ * Unlike UMMG-JSON, Echo10XML is sensitive to key ordering as it uses <sequence> in the schema.
```

Comment:
Minor nit - I'm not sure comparing it to UMMG adds useful information here
```suggestion
 * Echo10XML is sensitive to key ordering as it uses <sequence> in the schema.
```

Reply:
Would it be helpful to have a summary here? 

e.g.

```
/**
 * This method does <thing> 
 *
 * Echo10XML is sensitive
```